pkg/sql/prepared_stmt.go

```diff
@@ -325,8 +325,14 @@ func (p *PreparedPortal) size(portalName string) int64 {
 	return int64(uintptr(len(portalName)) + unsafe.Sizeof(p))
 }
 
+// isPausable checks if a portal is pausable.
+//
+// Note: We need this to be function rather than a static bool, because a
```

Comment:
This note doens't really make sense any longer - you can remove it.

Reply:
Sound good I just removed it, thanks Marcus!
<!-- Reviewable comment -OCiJbFqBB-BngK4HgrZ:b-tix1uy -->
<!-- Sent from Reviewable.io -->